test/Microsoft.TestPlatform.PerformanceTests/ProtocolV2Tests.cs

```diff
@@ -49,76 +52,79 @@ public class ProtocolV2Tests
     [TestMethod]
     public void TestCaseSerialize2()
     {
-        Serialize(TestCase, 2);
+        SerializeV2(TestCase);
         Stopwatch sw = Stopwatch.StartNew();
         for (int i = 0; i < 10000; i++)
         {
-            Serialize(TestCase, 2);
+            SerializeV2(TestCase);
         }
         sw.Stop();
 
-        VerifyPerformanceResult("TestCaseSerialize2", 2000, sw.ElapsedMilliseconds);
+        var actualDuration = sw.Elapsed;
+        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), $"when serializing 10k test cases");
     }
 
     [TestMethod]
-    public void TestCaseDeserialize2()
+    public void TestCaseDeserializeV22()
     {
-        var json = Serialize(TestCase, 2);
-        Deserialize<TestCase>(json, 2);
+        var json = SerializeV2(TestCase);
+        DeserializeV2<TestCase>(json);
 
         Stopwatch sw = Stopwatch.StartNew();
         for (int i = 0; i < 10000; i++)
         {
-            Deserialize<TestCase>(json, 2);
+            DeserializeV2<TestCase>(json);
         }
         sw.Stop();
 
-        VerifyPerformanceResult("TestCaseDeserialize2", 2000, sw.ElapsedMilliseconds);
+        var actualDuration = sw.Elapsed;
+        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), $"when de-serializing 10k test cases");
     }
 
     [TestMethod]
     public void TestResultSerialize2()
     {
-        Serialize(TestResult, 2);
+        SerializeV2(TestResult);
         Stopwatch sw = Stopwatch.StartNew();
         for (int i = 0; i < 10000; i++)
         {
-            Serialize(TestResult, 2);
+            SerializeV2(TestResult);
         }
         sw.Stop();
 
-        VerifyPerformanceResult("TestResultSerialize2", 2000, sw.ElapsedMilliseconds);
+        var actualDuration = sw.Elapsed;
+        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), $"when serializing 10k test results");
```

Comment:
Nit: interpolation is not needed.
```suggestion
        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), "when serializing 10k test results");
```